doc/user_download_and_install_windows.rst.template

```diff
@@ -0,0 +1,355 @@
+build LibXC, which will work on most systems:
+
+.. code-block:: bash
+
+    ${custom_install_libxc}
+
+This results in a LibXC library suitable for compiling HORTON. If this fails,
+consult your local Linux guru to build LibXC. For more info about LibXC, check
+the website: http://www.tddft.org/programs/octopus/wiki/index.php/Libxc
+
+**LibInt2**
+
+The directory ``tooals/qa`` of the HORTON source tree contains scripts to download and
+build LibInt2, which will work on most systems:
+
+.. code-block:: bash
+
+    ${custom_install_libint}
+
+The compilation of LibInt2 takes a few minutes and results in a library suitable for
+compiling HORTON. If this fails, consult your local Linux guru to build LibInt2. For more
+info about LibInt2, check the website: http://sourceforge.net/p/libint/home
+
+**Python dependencies**
+
+In some cases, PIP, the Python package manager, may be a good choice to install
+the most recent versions of the Python packages in the list of dependencies. Examples
+are given above for Ubuntu and Fedora.
+
+
+.. _linux_compile_install:
+
+Compilation and installation
+============================
+
+Build and install
+-----------------
+
+The regular build and install is done as follows:
+
+.. code-block:: bash
+
+    ./setup.py install --user
+
+The ``setup.py`` script makes a reasonable attempt at configuring the compiler and
+linker settings for the LibXC, LibInt2 and BLAS libraries. However, this does
+not work in all environments. In case of a failure, or if a configuration other
+than the default is desired, read the following section.
+
+
+Overriding default compiler/linker settings for LibXC, LibInt2 and BLAS
+-----------------------------------------------------------------------
+
+The manual configuration of the compiler and linker settings is described here:
+:ref:`setup_cfg`. You should read this section if the default build and install
+has failed or if you would like to specify which libraries to use.
+
+
+Runtime configuration
+---------------------
+
+You need to set the following variable in ``~/.bashrc`` to use HORTON:
+
+.. code-block:: bash
+
+    export PATH=${HOME}/.local/bin:${PATH}
+
+    # If you used special link options for LibXC, LibInt2 or BLAS, something along
+    # the following lines may also be needed:
+    # export LD_LIBRARY_PATH=some_dir/with/shared_objects/${LD_LIBRARY_PATH}
+
+If you run HORTON on a headless node, i.e. without an X server, you need to
+configure Matplotlib to use a backend that does not require a graphical user
+interface. (See http://matplotlib.org/faq/usage_faq.html#what-is-a-backend for
+more details on the Matplotlib backends.) This can be done by adding the
+following line to your ``matplotlibrc`` file:
+
+.. code-block:: text
+
+    backend: agg
+
+This file is located in either ``${HOME}/.matplotlib`` or
+``${HOME}/.config/matplotlib``.
+
+
+Compilation and Installation For Developers
```

Comment:
Redundant w.r.t. developer documentation. Please, add missing parts to the developer documentation and remove here.